topic04-git-introduction/book-git-tutorial/01.Basics.md

```diff
@@ -0,0 +1,46 @@
+---
+
+
+# Git guide
+
+
+This manual contains necessary commands for git when working on your own, contributing to other projects and working as a team.
+
+
+## What is git about?
+
+
+Git is a version control system that allows you to manage your code easily while working as a team or contributing to other repos. It allows you to merge code without having to copy and paste it from another file or files. Git also is able to host your website but it does not give you a proper full domain. 

```

Comment:
Git -> Github is able to host your website. Remember Git and Github are different things.